xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -239,6 +245,12 @@ class CPVRRecording final : public CVideoInfoTag
    */
   bool IsRadio() const { return m_bRadio; }
 
+  /*!
+   * @brief Retrieve the type of media this recording is
+   * @return PVR_RECORDING_MEDIA_TYPE_RECORDING if this is a pvr recording, PVR_RECORDING_MEDIA_TYPE_VOD if this is a tv recording
```

Comment:
What about PVR_RECORDING_MEDIA_TYPE_UNKNOWN ?

Reply:
Not relevenat, media type will always be known.